docs/release-notes/next/PT-165563336_SC_generic_messages_in_every_state.md

```diff
@@ -0,0 +1,4 @@
+* Allows sending and receiving generic messages in State Channels in any FSM
+  state. Until this - generic messages were allowed only in `open` state.
+  `channel_id` is part of the message body and if it is unknown - the
+  temporary one is used instead. This is breaking off-chain consesus.
```

Comment:
It is unclear to me what it is meant as "off-chain consensus".

Based on what I see in code, the items affected are called "inband messages" and I see them defined as part of the protocol  inside the channel Noise endpoint among two nodes, as [message code 96](https://github.com/aeternity/protocol/blob/3a7b1bdf1b2fa9dee98ac0788ca084b654912c84/channels/OFF-CHAIN.md#inband_message):
> Inband messages are arbitrary messages sent between the channel participants. The payload must be limited to 65,535 bytes. The fsm must deliver an inband message immediately, and the receiver must process it (e.g. conveying it to the client) immediately, preserving the message ordering.
>
> One possible use of inband messages could be to synchronize off-chain state updates, but any application is allowed.

Is what is broken by this change:
- The protocol in the channels Noise endpoint among two nodes; plus
- The user channels WebSocket API

?

Reply:
> The protocol in the channels Noise endpoint among two nodes; plus

This is broken. This also is the off-chain (noise) protocol between the nodes/FSMs

> The user channels WebSocket API

This is not broken but enhanced - consumers of the WebSocket API are now able to send and receive messages while the channel is being updated.